src/pages-conditional/developers/learning-tools.js

```diff
@@ -175,6 +175,7 @@ const LearningToolsPage = ({ data }) => {
         <StyledCardGrid>
           {sandboxes.map((sandbox, idx) => {
             return (
+              /* Product Name: Do Not Translate name/alt */
```

Comment:
why are we not translating alt?

Reply:
I suspect because they're all product/company names?

Reply:
An alt description that is just a product name isn't very descriptive. At the least it'd be a logo I imagine in which case I think it needs a string.

Reply:
This page for example (`learning-tools.js`) has the "alt" text hard coded into the product data array which is simply the product name (ie. `alt: "Remix",`).. Others use string interpolation to add " logo", like the `dapps.js` page. I didn't change any of the existing terms, just tried to update what was already there as it was, which is why this did not have any translation for "alt"... because "alt" was just "Remix" or "Ethernauts" for example.

I do think some standardization one way or the other would help. And I kinda agree that, given the purpose of "alt", especially for visually impaired users who require text readers, having the word "logo" _does_ give more context for the user. This of course would come with the trade-off of more lines to translate, though I doubt it would be all that difficult to translate "logo" a few times (by having "Uniswap logo", "Ethernaut logo", "Remix logo" etc, as unfragmented language strings)